doc/source/whatsnew/v2.0.2.rst

```diff
@@ -0,0 +1,41 @@
+.. _whatsnew_202:
+
+What's new in 2.0.2 (May ..., 2023)
+-----------------------------------
+
+These are the changes in pandas 2.0.2. See :ref:`release` for a full changelog
+including other versions of pandas.
+
+{{ header }}
+
+.. ---------------------------------------------------------------------------
+.. _whatsnew_202.regressions:
+
+Fixed regressions
+~~~~~~~~~~~~~~~~~
+- Fixed regression when :meth:`DataFrame.to_string` creates extra space for string dtypes (:issue:`52690`)
```

Comment:
Could you merge in main and add this to the existing `v2.0.2.rst` file instead?

Reply:
ok, should have done this